src/assets/_project/_blocks/legacy/forms/_old_forms.scss

```diff
@@ -876,7 +876,7 @@ form button,
   max-width: none;
   font-weight: bold;
   font-style: normal;
-  color: #ffffff;
+  color: white;
```

Comment:
use of a variable may be even better.

Reply:
I don't see why we would define our own 'white' variable when one is built in.

Reply:
In the future, we may want to make white a slight grey white. since we already have all colors as variables (or soon will). Might as well as white and black in the palette config file.

Reply:
I've added a definition for the 73% intensity white that's in use.